docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -0,0 +1,282 @@
+==============================================================
+Upgrade Guide - Scylla Monitoring 3.x to Scylla Monitoring 4.y
+==============================================================
+
+This document is a step by step procedure for upgrading `Scylla Monitoring Stack </operating-scylla/monitoring/3.0>`_ from version 3.x to 4.y, for example, between 3.9 to 4.0.0.
+
+Upgrade Procedure
+=================
+
+We recommend installing the new release next to the old one. You can run both monitoring stacks in parallel, ensuring it is working as expected before uninstalling the old version.
+
+Change to the directory you want to install the new Monitoring stack.
+Download the latest release in the .zip or .tar format.
+
+Install 4.y (The new version)
+-----------------------------
+
+.. code-block:: bash
+
+                wget -L https://github.com/scylladb/scylla-monitoring/archive/scylla-monitoring-4.y.zip
+                unzip scylla-monitoring-4.y.zip
+                cd scylla-monitoring-scylla-monitoring-4.y/
+
+Replace “y” with the new minor and patch release number, for example, 4.0.0.zip
+
+Setting the server's files
+--------------------------
+
+Copy the ``scylla_servers.yml`` and ``scylla_manager_servers.yml`` from the version that is already installed.
+
+.. code-block:: bash
+
+                cp /path/to/monitoring/3.x/prometheus/scylla_servers.yml prometheus/
+                cp /path/to/monitoring/3.x/prometheus/scylla_manager_servers.yml.yml prometheus/
+
+Validate the new version is running the correct version
+-------------------------------------------------------
+
+run:
```

Comment:
```suggestion
Run:
```

Reply:
lol, you changed it from Run to run in the previous review